modules/push/src/main/WebSubscriptionApi.scala

```diff
@@ -23,8 +25,13 @@ final class WebSubscriptionApi(coll: Coll)(using Executor):
       )
       .void
 
-  def unsubscribeBySession(sessionId: String): Funit =
-    coll.delete.one($id(sessionId)).void
+  def unsubscribeBySession(id: SessionId | AccessTokenId): Funit =
+    coll.delete.one($id(subscriptionId(id))).void
+
+  private def subscriptionId(id: SessionId | AccessTokenId): String =
+    id match
+      case sessionId: SessionId         => sessionId.value
+      case accessTokenId: AccessTokenId => accessTokenId.value
```

Comment:
unfortunately the match won't work because at runtime they're both `String`.

Reply:
Uff, so before it was always just taking the first branch of the match, which just happened to be working 😱